webapp/sources/rudder/rudder-web/src/main/scala/com/normation/rudder/web/components/RuleEditForm.scala

```diff
@@ -527,17 +527,17 @@ class RuleEditForm(
     val dirTags        = net.liftweb.json.compactRender(JsonTagSerialisation.serializeTags(d.tags))
     JsRaw(
       s"""onClickDirective("${dirId}", ${dirName}, ${dirLink}, ${dirDescription}, ${dirTechName}, ${dirTechVersion}, ${dirMode}, ${dirTags})"""
-    )
+    ) // JsRaw ok, escaped
   }
 
   private[this] def includeRuleTarget(targetInfo: FullRuleTargetInfo): JsCmd = {
-    val target = targetInfo.target.target.target
-    JsRaw(s"""includeTarget(event, "${target}");""")
+    val target = targetInfo.target.target.target.encJs
+    JsRaw(s"""includeTarget(event, "${target}");""") // JsRaw ok, escaped
   }
 
   private[this] def excludeRuleTarget(targetInfo: FullRuleTargetInfo): JsCmd = {
-    val target = targetInfo.target.target.target
-    JsRaw(s"""excludeTarget(event, "${target}");""")
+    val target = targetInfo.target.target.target.encJs
+    JsRaw(s"""excludeTarget(event, "${target}");""") // JsRaw ok, escaped
```

Comment:
I still think it is possible to get "unescaped" targets, so I wouldn't have considered these `JsRaw`s to be escaped yet... :confused: